### PR TITLE
Update augeas to 1.11.0

### PR DESCRIPTION
**Pull request by grueni**
Lenses delivered with th package should be stored in /usr/share/augeas/lenses/dist and site specific lenses /usr/share/augeas/lenses.

### CODE REVIEW

**Review by AndWac — APPROVED** [review]

**Comment by pyhalov** [comment]
Is this update ABI-compatible? If not, have you tested nut  and ruby/augeas build against this version?

**Comment by grueni** [comment]
No changes in ABI.
There are additional features in the lenses and new lenses.

**Comment by Mno-hime** [comment]
It should be: https://abi-laboratory.pro/index.php?view=timeline&l=augeas.

File: components/sysutils/augeas/augeas-lenses.p5m [hunk 1]
```diff
@@ -22,222 +23,224 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
...
+file path=usr/share/augeas/lenses/dist/wine.aug
+file path=usr/share/augeas/lenses/dist/xendconfsxp.aug
+file path=usr/share/augeas/lenses/dist/xinetd.aug
+file path=usr/share/augeas/lenses/dist/xml.aug
+file path=usr/share/augeas/lenses/dist/xorg.aug
+file path=usr/share/augeas/lenses/dist/xymon.aug
+file path=usr/share/augeas/lenses/dist/xymon_alerting.aug
+file path=usr/share/augeas/lenses/dist/yaml.aug
+file path=usr/share/augeas/lenses/dist/yum.aug
+
```
Comment (pyhalov) [inline components/sysutils/augeas/augeas-lenses.p5m]: Please, remove empty line.

Comment (grueni) [reply components/sysutils/augeas/augeas-lenses.p5m]: @Michal Thanks for the link I didn't know it.

**Pull request merged**